include/swift/AST/Attr.def

```diff
@@ -733,6 +733,11 @@ CONTEXTUAL_SIMPLE_DECL_ATTR(_local, KnownToBeLocal,
   APIBreakingToAdd | APIBreakingToRemove,
   130)
 
+SIMPLE_DECL_ATTR(ignoreDeprecationWarnings, IgnoreDeprecationWarnings,
+  OnAbstractFunction | OnGenericType | OnVar | OnSubscript | OnEnumElement | OnImport |
+  OnExtension | AllowMultipleAttributes |
```

Comment:
Just noticed you marked this `AllowMultipleAttributes`, note that it means that multiple of the same attribute is allowed, e.g it allows `@ignoreDeprecationWarnings @ignoreDeprecationWarnings func`. It doesn't affect mixing with other attributes. I think it probably shouldn't be marked `AllowMultipleAttributes`, what do you think?